serverless.yml

```diff
@@ -130,6 +132,11 @@ resources:
       Properties:
         QueueName: aqts-capture-error-queue-${self:provider.stage}
         MessageRetentionPeriod: 1209600
+    terminalErrorsQueue:
```

Comment:
I wonder if a event needs to be added to the trigger as a sibling PR to this one. That event would be disabled be default. :thinking:

Reply:
I'm not following on this.  This PR is just trying to throw the dead message in the queue, where it will sit for 4 days so the dashboard can display it.  I'm thinking SNS is still the primary mechanism for dealing with terminal errors.  Not sure what the event would do?

Reply:
I think the intent is that SQS could also be used to feed payloads back into the state machine via the trigger. That however can be a separate ticket.